pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/nativefst/NativeFSTIndexCreator.java

```diff
@@ -21,8 +21,11 @@
 import java.io.File;
```

Comment:
Is there a reason this class does not extend AbstractTextIndexCreator.java?

Reply:
Reason is explained in https://github.com/apache/pinot/pull/10184#discussion_r1138794547